test/foo/lib/config/test_configloader.py

```diff
@@ -0,0 +1,55 @@
+from unittest.mock import Mock
+
+import pytest
+
+from foo.lib.config import ConfigLoader, ConfigLoaderException
+
+
+def reset_config_loader_singleton():
```

Comment:
seems like this method is used in every test case so would be handly to include it in the test setup method
I think it's called fixtures for pytest?